### PR TITLE
Feature/server/124 add execution notification into game

### PR DESCRIPTION
**Pull request by YIllmann**
Adds notification endpoints to post and retrieve notification from the web api to the mobile player devices. 
The get request is limited to the Mobile API and the Post endpoint is limited to the web-api.

Further a setup module for execution.web has been added.

### CODE REVIEW

File: server/app.py [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
+
 from flask import Flask, send_from_directory, redirect, request
 from flask_jwt_extended import JWTManager, jwt_required
 from flask_sqlalchemy import SQLAlchemy
 from flask_wtf.csrf import CSRFProtect
 from werkzeug.exceptions import BadRequestKeyError
-
-from execution.api import location, patient, actions
 from execution.utils import util
 from execution.entities.execution import Execution
```
Comment (simohlsen) [inline server/app.py:10]: By convention, imports are grouped by standard libraries, external libraries and internal imports, no need to change that here.

**Review by simohlsen — CHANGES_REQUESTED** [review]

File: server/app.py (hunk 1 above)
Comment (YIllmann) [reply server/app.py:10]: this import moved to line 24 as asynch import

File: doc/api/api.yaml [hunk 2]
```diff
@@ -406,14 +406,12 @@ paths:
       summary: notification polling
       description: Returns a list of notifications for the user after lastPollTime.
       parameters:
-        - name: lastPollTime
+        - name: next_id
```
[the remark this answers is not in the document]
Comment (YIllmann) [reply doc/api/api.yaml:409]: is implemented as suggested. Next_id points to the next notification the player would receive and is bounded by the length of the list.

**Review by simohlsen — APPROVED** [review]

File: server/execution/api/notification.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
+from flask import Blueprint, request
+from flask_jwt_extended import jwt_required
+from execution.utils import util
+
+api = Blueprint("api-notification", __name__)
+
+
+@api.get("/notifications")
+@jwt_required()
+def get_notification():
```
Comment (7erra) [inline server/execution/api/notification.py:10]: Function name should be plural

File: server/execution/web/notification.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+@web_api.post("/notifications/post")
+def add_notification_to_execution():
+    """
+    Endpoint for adding a notification to an execution via the execution id.
+    Each parameter is set as form parameter.
+    Returns OK for successful requests or BAD_REQUEST in case of invalid parameters
+    """
+    form = request.form
+
+    try:
```
Comment (7erra) [inline server/execution/web/notification.py:19]: After #125 is merged you can use the `@required` decorator which handles form and query parameter validation.

**Review by 7erra — CHANGES_REQUESTED** [review]

**Review by 7erra — APPROVED** [review]

**Pull request merged**